src/meta/src/hummock/manager/commit_epoch.rs

```diff
@@ -65,7 +65,6 @@ pub struct CommitEpochInfo {
     pub change_log_delta: HashMap<TableId, ChangeLogDelta>,
     pub committed_epoch: u64,
     pub tables_to_commit: HashSet<TableId>,
-    pub is_visible_table_committed_epoch: bool,
 }
 
 impl HummockManager {
```

Comment:
The documentation below is inaccuarte, right? IIUC commit_epoch can take a fake epoch as well.